kwanj-k/ride-my-way
# 158529322 -User well formatted/informative links

## What does the pull request do?
Fixes the links format

## Features
User finds well formatted links on navigation


### Testing

 * Clone this repository to a local folder
    ``` 
    git clone https://github.com/kwanj-k/ride-my-way.git 
    ```
 * Navigate to the cloned repo
 * Run ``` git checkout bg-user-well-formatted-links-158529322  ```
 * Open index.html using your favorite browser
 * Go through the UI to see the changed links

### Pivotal Tracker Board [click here](https://www.pivotaltracker.com/story/show/158529322#)
